Archer V2 search records and fetch incidents raw XML filtering (#37615)
demisto/content: Packs/ArcherRSA/ReleaseNotes/1_3_0.md

#### Integrations

##### RSA Archer v2

> - Added a new "xmlForFiltering" argument to the ***archer-search-records*** command to allow for the filtering of records using a raw XML condition element such as: "DateComparisonFilterCondition" or "TextFilterCondition".

Reviewer: ```suggestion
- Added a new "xmlForFiltering" argument to the ***archer-search-records*** command to allow filtering of records using a raw XML condition element such as: "DateComparisonFilterCondition" or "TextFilterCondition".
```

#### Integrations

##### RSA Archer v2

- Added a new "xmlForFiltering" argument to the ***archer-search-records*** command to allow for the filtering of records using a raw XML condition element such as: "DateComparisonFilterCondition" or "TextFilterCondition".
- Added new "XML for fetch filtering" and "Fetch filtering logic operator" configuration parameters that affect which incidents are fetched.

>   - The "XML for fetch filtering" configuration parameter cannot be a "DateComparisonFilterCondition" XML element since it would interfere with the existing fetch date filter.

Author: Removed from release notes (sentences appear in the integration `README.md`)

##### RSA Archer v2

- Added a new "xmlForFiltering" argument to the ***archer-search-records*** command to allow for the filtering of records using a raw XML condition element such as: "DateComparisonFilterCondition" or "TextFilterCondition".

> - Added new "XML for fetch filtering" configuration parameter that affect which incidents are fetched.

Reviewer: ```suggestion
- Added new "XML for fetch filtering" configuration parameter that allows advanced filtering of incidents.```
